config/source/cli/plugins.go

```diff
@@ -0,0 +1,32 @@
+// Package cli provides cli.
+package cli
+
+import (
+	"errors"
+
+	"go-micro.dev/v5/log"
+	"go-micro.dev/v5/util/container"
+)
+
+var (
+	// ErrFlagExists is returned when the given element exists in the flag container.
+	ErrFlagExists = errors.New("element exists already")
+
+	// Plugins contains source/cli subplugins, for example urfave, pflag, cobra.
+	Plugins = container.NewMap[ParseFunc]() //nolint:gochecknoglobals
+	// Flags is the global flag container where you have to register flags with.
+	Flags = container.NewList[*Flag]() //nolint:gochecknoglobals
+)
+
+func init() {
+	flag := NewFlag(
+		"logger",
+		log.DefaultPlugin,
+		CPSlice([]string{"logger", "plugin"}),
+		Usage("Default logger to use, jsonstderr, jsonstdout, textstderr, textsdout."),
```

Comment:
I think this depends on the plugins the user imports no? so the defaults suggestions are wrong if the user doesn't impor those

Reply:
They are just here to let the user know "what are some values" that maybe could happen here, on the other side you'r right yeah. Not sure which is better.

In go-micro.dev/v4 we have lots of suggestions here (logrus, zap and so on).

Reply:
Then maybe something like `"Default logger to use (e.g. jsonstderr, jsonstdout, textstderr, textsdout)."`

Reply:
Sounds good.